mintapi/api.py

```diff
@@ -35,6 +36,10 @@
 except ImportError:
     pd = None
 
+logger = logging.getLogger('mintapi')
+logger.addHandler(logging.StreamHandler())
+logger.setLevel(logging.INFO)
```

Comment:
Nice logging changes, thanks!

Reply:
Sure thing! I wanted to add new lines for the chromedriver downloading bit, but only wanted to do so if clients had the option to change the log output. Using the standard logging module allows clients to grab a reference to the 'mintapi' logger, and change the level or routing in their own code.